business/services.go

```diff
@@ -143,14 +143,14 @@ func (in *SvcService) GetServiceApiDocumentation(namespace, service string) (str
 
 	resp, err2 := http.Get(apiSpecPath)
 	if err2 != nil {
-		log.Errorf("GET error: %v", err)
+		log.Errorf("API Documentation GET error: %v", err)
```

Comment:
why is there a need to hardcode "GET"?

Reply:
I was keeping the old message, just adding context. Now rewritten to give even more info